doc/_admin-guide/070_Destinations/230_Slack/README.md

```diff
@@ -48,8 +48,8 @@ You can find the source of the Slack configuration snippet on GitHub.
 
 ## Prerequisites
 
-To send messages and notifications from syslog-ng OSE to Slack, you must
-create a Slack app and a Webhook that syslog-ng OSE can use. For
+To send messages and notifications from {{ site.product.short_name }} to Slack, you must
+create a Slack app and a Webhook that {{ site.product.short_name }} can use. For
 details, see the [[Slack documentation|slack-webhook-doc]].
```

Comment:
Apart from this markdown link is not needed here `Slack documentation` is already an external link with the id slack-webhook-doc, but can we use instead of here and in the external link, the `Slack webhook documentation` sentence